mw-webapp/src/component/footer/Footer.module.scss

```diff
@@ -1,52 +1,72 @@
 @import "src/styles/variables";
 
-$userNameFontSize: 16px;
+$logoHeight: 55px;
+$iconSize: 32px;
+$privacyLineHeight: 140%;
+$socialTextFontWeight: 500;
+$rulerHeight: 1px;
 
 .footer {
   background-color: var(--quaternaryBgColor);
 }
 
 .footerWrapper {
-  width: 100%;
   max-width: $maxWidthContainer;
-  justify-content: space-between;
-  padding: $paddingBig $paddingMedium;
-  border-bottom: $primaryBorder solid var(--secondaryStrokeColor);
+  box-sizing: content-box;
+  flex-direction: column;
+  padding: 0 $paddingMedium;
   margin: 0 auto;
+  gap: 0;
```

Comment:
if gap 0 then you can delete property

Reply:
I can't because horizontalContainer has gap:8px by default, and I don't need this value